Gruntfile.js

```diff
@@ -1,70 +1,64 @@
 module.exports = function(grunt) {
 
-	grunt.initConfig({
-		pkg: grunt.file.readJSON('package.json'),
-		jshint: {
-			files: ['Gruntfile.js', 'src/**/*.js', 'tests/**/*.js'],
-			options: {
+  grunt.initConfig({
+    pkg: grunt.file.readJSON('package.json'),
+    // Code formatting / linting
+    jscs: {
+      src: 'src/hello.js',
```

Comment:
Sounds great! If you don't mind returning the Gruntfile back to tabs I think we can merge the changes thus. Or if you'd prefer it to be in one PR, up to you.